### DISCUSSION TITLE
edit email body for new order

### DISCUSSION THREAD

**Discussion by alemao8** [pretix/pretix#2881] [Q&A]
Hello.

I am trying to edit the body of the email that is sent when a new paid order is placed, in two languages.
Is there a way to edit this?
I have come to a conclusion (so far) that the email template "placed order" is not the email that is sent when a paid order is placed.
I receive an email that contains the phrase "Please transfer the full amount to the following bank account.", but this phrase is not included in the placed order template email.

The installation is a pretix community.

Thank you in advance

**Comment by raphaelm** [comment]
> I have come to a conclusion (so far) that the email template "placed order" is not the email that is sent when a paid order is placed.

Yes it is :)

> I receive an email that contains the phrase "Please transfer the full amount to the following bank account.", but this phrase is not included in the placed order template email.


The template probably includes the payment_info tag which dynamically inserts content based on the payment method used.

**Comment by alemao8** [comment]
Thank you for the clarifications.

The issue I am facing seems to be with the {payment_info} placeholder itself in different languages.
I am attaching 3 screenshots where you can see that the style of the info placed inside {payment_info} is different in greek compared to english and german translations.

english: https://prnt.sc/8xxYyDKj2MBT
german: https://prnt.sc/nx8dftEB1a4n
greek: https://prnt.sc/j9drfNtccwVk

How can we fix this ?

Thank you in advance.

**Comment by wiffbi** [comment]
It was a problem in the formatting of the greek translation. We fixed it and it will be deployed soon.

**Comment by alemao8** [comment]
Hello and thank you for your immediate response.
Is there something that has to be done from our side in the current implementation ? Perhaps editting/replacing some files ?

Thank you in advance

**Reply by wiffbi** [reply]
If you host pretix yourself, iirc you probably need to wait until the next release end of this month and update. I am sorry!

**Reply by alemao8** [reply]
Yes this is the case, we are hosting pretix. No worries this is not urgent.
If this is convenient for you, please let me know which file/files we can edit in order to solve this in the meanwhile.

Highly appreciated !!
